devices/sonoff/snzb-06p.json

```diff
@@ -0,0 +1,126 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "SONOFF",
+  "modelid": "SNZB-06P",
+  "vendor": "Sonoff",
+  "product": "SNZB-06P",
+  "sleeper": false,
+  "status": "Gold",
+  "path": "/devices/snzb-06p.json",
+  "subdevices": [
+    {
+      "type": "$TYPE_PRESENCE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0406"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/dark",
+          "refresh.interval": 300,
+          "read": {
+            "at": "0x2001",
+            "cl": "0xfc11",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x2001",
+            "cl": "0xfc11",
+            "ep": 1,
+            "eval": "if (Attr.val === 0) { Item.val = true } else { Item.val = false }",
+            "fn": "zcl:attr"
+          },
+          "default": false
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/presence",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0406",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": false
+        }
+      ]
+    }
+  ],
+  "bindings": [
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0x0500"
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0x0406",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x18",
+          "min": 1,
+          "max": 300,
+          "change": "0x00000001"
```

Comment:
As the datatype is discrete here, `"change": "0x00000001"` is superfluous. Therefore, please remove that line and do not forget about the comma one line above 🙂